keras_cv/layers/preprocessing/random_resized_crop.py

```diff
@@ -0,0 +1,140 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import warnings
+
+import tensorflow as tf
+
+from keras_cv.layers import BaseImageAugmentationLayer
+from keras_cv.utils import preprocessing
+
+
+@tf.keras.utils.register_keras_serializable(package="keras_cv")
+class RandomResizedCrop(BaseImageAugmentationLayer):
+    """
+    Randomly crops a part of an image and resizes it to provided size.
+
+    Args:
+        target_size: A tuple of two integers used as the target size to crop
+            images to.
+        aspect_ratio_factor: (Optional) A tuple of two floats. Represents the
+            lower and upper bounds for the aspect ratio of the cropped image
```

Comment:
Maybe clarify what the aspect ratio mean here?

Reply:
good point, IIUC it is width/height